src/content/quickstart/openshift/_index.md

```diff
@@ -4,9 +4,30 @@ title: "OpenShift (AWS)"
 weight: 20
 ---
 
+This quickstart guide covers the necessary steps to deploy two OpenShift Container Platform (OCP) clusters on AWS with full stack automation
+(aka IPI). Once the OpenShift clusters are deployed, we will walk you through the deployment of Submariner,
+interconnecting the two clusters. Note that this guide focuses on Submariner for network connectivity only. For Submariner with
+Service Discovery, please refer to [this guide](./service_discovery/).
+
 {{< include "quickstart/openshift/setup_openshift.md" >}}
+
+{{% notice info %}}
+Please ensure that the tools you downloaded above are compatible with your OpenShift Container Platform version. For more information,
+please refer to the official [OpenShift documentation](https://docs.openshift.com/container-platform/).
+{{% /notice %}}
+
+{{< include "quickstart/openshift/setup_aws.md" >}}
 {{< include "quickstart/openshift/create_clusters.md" >}}
-{{< include "quickstart/openshift/ready_clusters.md" >}}
+
+### Prepare Your AWS Clusters for Submariner
+
+{{< include "quickstart/openshift/download_prep_for_subm.md" >}}
+
+{{% notice info %}}
+Please note that  **oc**, **aws-cli**, **terraform**, and **wget** need to be installed before the `prep_for_subm.sh` script can be run.
```

Comment:
I'm not sure about bold for these tools. The K8s guide only says to use bold for UI elements, and we additionally use bold to highlight key phrases. Maybe these tools are key phrases, but I could also see them following the K8s command tool (`kubectl`) guidance.

https://kubernetes.io/docs/contribute/style/style-guide/#use-code-style-for-kubernetes-command-tool-and-component-names